src/Messaging/test/Benchmarks/Channel/BenchmarkDotNet.Artifacts/results/Program-report-github.md

```diff
@@ -0,0 +1,14 @@
+``` ini
```

Comment:
Should benchmark artifacts be included?

Reply:
I think its worth having a record/trail of the benchmark performance for historical reasons.  I think it records the processor it was run on, etc... so might be helpful in the future...
But, I'm not hard set on it..